iina/FFmpegController.mm

```diff
@@ -0,0 +1,205 @@
+//
+//  FFmpegController.mm
+//  iina
+//
+//  Created by Saagar Jha on 2/16/19.
+//  Copyright © 2019 lhc. All rights reserved.
+//
+
+#import "FFmpegController.h"
+
+// Perhaps this will work someday? Until then, we need to keep around our dummy
+// interface in the header file.
+//#import "IINA-Swift.h"
```

Comment:
Does it work now?😊

Reply:
I put a bit of effort into this and I just don't think it was worth it. A lot of things break and need to be subtly tweaked so the Swift importer is happy and I just gave up after disabling modules, forward-declaring a bunch of private APIs and moving a bunch of code around didn't fix the issue.